### ISSUE TITLE
Integrate EBAM data with RPI-Monitor

### ISSUE THREAD

**Issue by patricktokeeffe** [patricktokeeffe/rpi-ebam-plus#2]
We're using [RPI-Monitor](rpi-experiences.blogspot.com) to visualize system health/history of the underlying hardware. It's pretty easy to add metrics to this system and we want to add

* new dynamic metrics corresponding to EBAM measurements
* new panel on Status page showing
  * latest measured value
  * latest measurement timestamp
  * bonus points for nice image
* new plots on Statistics page for EBAM measurements

The configuration should be encapsulated in a new RPI-Monitor config file. [See examples here](https://xavierberger.github.io/RPi-Monitor-docs/31_configuration_examples.html)

----

To actually access EBAM measurement values, use this two-step approach:
1. data retrieval script writes measurement values into individual files located in RAM

```
/run/ebam/pm25
```
```
10.3
```

2. use `cat` to read values from those same files by specifying source like so: 
```
...
dynamic.N.source=cat /run/ebam/pm25
...
```

**Comment by patricktokeeffe** [comment]
To keep EBAM-specific config separated, put entries into new file `/etc/rpimonitor/template/ebam.conf` and add that file to include list at bottom of `/etc/rpimonitor/data.conf`:
```
...
include=/etc/rpimonitor/template/ebam.conf

```

**Comment by patricktokeeffe** [comment]
Closed by 8e00f714450c964333bd44d61d704a2258c50718
